Makefile

```diff
@@ -12,6 +21,25 @@ useCompositeBuild=true
 dry_run=false
 instrumentation=Ui
 stacktrace?=
+project=-p subprojects
+
+docker_command?=
+
+ifeq ($(docker),true)
+define docker_command
+make clear_gradle_lock_files
+make clear_docker_containers
+docker run --rm \
+	--volume "$(shell pwd)":/app \
+	--volume "$(GRADLE_CACHE_DIR)":/gradle/caches \
+	--volume "$(GRADLE_WRAPPER_DIR)":/gradle/wrapper \
+	--workdir /app \
+	--env TZ="Europe/Moscow" \
+	--env LOCAL_USER_ID="$(USER_ID)" \
+	--env GRADLE_USER_HOME=/gradle \
+	android-builder
+endef
+endif
 
```

Comment:
About full migration to makefile:

I'd like to discuss such decisions in a more formal way. I see a change, but don't understand the whole picture.
I'm trying to figure out PRO/CON and compare (example: https://medium.com/redbubble/rewriting-our-deploy-tooling-from-makefile-to-bash-and-back-again-4edeab5b0afa). 
Will we do this in an internal repository?

Reply:
I like to see this repo as a platform for more experiments, successful ones could be used in internal, this one could too.
Do you like to see a discussion about makefile first? With more bold points?

Reply:
> Do you like to see a discussion about makefile first? With more bold points?

Yes, it would be helpful to write and share knowledge about such experiments. 
Ideally, I want to manage them as some artifacts besides the code. Maybe only in issues. For big changes - tech specs, ...
When something is only in code, it's harder to see common points: 
- What experiments we have and why? 
- In what state?
- How can I help?

The change itself is good. I'm talking only about making the process more transparent. It's like the idea from Kanban - to visualize all unfinished work

Reply:
I will merge this one as it only adds value: local makefile commands in docker
And not changes bash entrypoints yet
Will start a discussion